node/cli/res/default_chain.json

```diff
@@ -1,5 +1,5 @@
 {
-  "balances": [["alice", 100], ["bob", 100], ["john", 10]],
-  "initial_authorities": ["alice"],
+  "accounts": [["alice", "6fgp5mkRgsTWfd5UWw1VwHbNLLDYeLxrxw3jrkCeXNWq", 100], ["bob", "6fgp5mkRgsTWfd5UWw1VwHbNLLDYeLxrxw3jrkCeXNWq", 100], ["john", "6fgp5mkRgsTWfd5UWw1VwHbNLLDYeLxrxw3jrkCeXNWq", 10]],
```

Comment:
what do you think about using objects for chain spec for this stuff? i think it would be useful to understand what's going on here, and we don't save much by having compact format..

i.e. {'accounts': [{'alias': 'alice', 'pub_key': '<>', 'balance': 100},... ]}

don't think we should change in this PR, just commenting because this is starting to get a little confusing